packages/foreman/rubygem-concurrent-ruby/rubygem-concurrent-ruby.spec

```diff
@@ -29,6 +29,7 @@ BuildArch: noarch
 
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 %{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+%{?scl:Obsoletes: rubygem-%{gem_name}}
```

Comment:
My concern with this is that it's not a direct replacement. If there's some other tool that does need the system Ruby version of this, it doesn't conflict. It does mean we may need a different strategy for removing the old files because yum doesn't handle leaf packages by default.

Reply:
I can not obsolete these and leave them. I don't know that I care that much. Folks shouldn't be using our packages with their custom tools IMO.